orc/src/main/java/org/apache/iceberg/orc/OrcFileAppender.java

```diff
@@ -164,14 +164,15 @@ private static Writer newOrcWriter(OutputFile file,
     }
 
     writer.addUserMetadata(COLUMN_NUMBERS_ATTRIBUTE, columnIds.serialize());
-    metadata.forEach((key,value) -> writer.addUserMetadata(key, ByteBuffer.wrap(value)));
+    metadata.forEach((key, value) -> writer.addUserMetadata(key, ByteBuffer.wrap(value)));
 
     return writer;
   }
 
   @SuppressWarnings("unchecked")
-  private static <D> OrcValueWriter<D> newOrcValueWriter(TypeDescription schema,
-                                                         Function<TypeDescription, OrcValueWriter<?>> createWriterFunc) {
+  private static <D> OrcValueWriter<D> newOrcValueWriter(
+      TypeDescription schema,
+      Function<TypeDescription, OrcValueWriter<?>> createWriterFunc) {
```

Comment:
I don't think this should have a newline. These should fit on one line, right?

Reply:
Yes! I'll update